### DISCUSSION TITLE
Specifying generators in package.json

### DISCUSSION THREAD

**Discussion by ChristianIvicevic** [prisma/prisma#18056] [Q&A]
I am using Prisma in a monorepo and have been using three generators now, the default JS client generator, Pothos and one for Rust. In my CI I have two parallel jobs for testing that run against all things TypeScript and all things Rust. Unfortunately when running `prisma generate` by default (especially in the postinstall hook) all generators are run forcing me to both install Node and Rust into the jobs even though I won't need both in both jobs.

As such I was curious whether it's possible to specificy the default generators to be run in the `package.json` in a similar fashion to how the schema path can be spcified in there. This would make it possible for me to list all JS related generators as a default to prevent running the Rust generator in the Node job.

Is this possible or planned?

**Comment by nurul3101** [comment]
Hello @ChristianIvicevic 👋 

I cannot find any existing feature request for specifying generators in package.json
Can you create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for it?

**Comment by janpio** [comment]
We recently introduced the `--generator` flag for `prisma generate`, but as you said this will not help with the postinstall hook at all. We could define that via `package.json`, as you suggested or also via an env var maybe. Feature request definitely welcome.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
